app-backend/api/src/main/resources/application.conf

```diff
@@ -50,6 +50,9 @@ auth0 {
 }
 
 s3 {
+  region = "us-east-1"
+  region = ${?AWS_DEFAULT_REGION}
+  dataBucket = ${?DATA_BUCKET}
```

Comment:
I added `DATA_BUCKET` to the task definition environment in https://github.com/azavea/raster-foundry-deployment/pull/39.